src/partials/device_profile_required.hbs

```diff
@@ -0,0 +1,14 @@
+<div class="card">
+  <div class="org-logo">
```

Comment:
There is no chance of errors happening here right? We just get DEVICE_PROFILE_REQUIRED and automatically submit the continue action 3 seconds later?

Reply:
At least for ID Dataweb IK it does work this way, if there is any error within the adapter it should be handled by the adapter's failure mode.

Not sure if you are worried about other kinds of error?

Reply:
Yeah I wasn't entirely sure what errors could occur during this flow. The widget will display them in the errors handlebar automatically if the {{> errors}} code is included.

I think if we feel confident that no errors would happen in this state then were fine

Reply:
I see, I'll add {{> errors}} in there since it looks like that is common for all the other _required handlebars (and just in case other intelligence kits handle errors differently).